### DISCUSSION TITLE
Nuclear depth decoy (cheap) incorrectly translated to "Politician" in chinese

### DISCUSSION THREAD

**Discussion by AzureNocte** [FakeFishGames/Barotrauma#13911] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

In current version, cheap version of nuclear depth charge and decoy are added, but in chinese translation, the nuclear depth decoy is translated into "政客", which means politician, where it should be "核能深潜诱饵 (廉价)".
I think it is caused by this small mistake in the translation file: SimplifiedChineseVanilla.xml, Line 8424

```xml
8422 <entityname.nucleardepthchargecheap>核能深水炸弹 (廉价)</entityname.nucleardepthchargecheap>
8423 <entitydescription.nucleardepthchargecheap>一桶超级炸弹。会先下潜然后爆破。由特定工程师廉价制造。</entitydescription.nucleardepthchargecheap>
8424 <entityname.nucleardepthdecoycheap>政客</entityname.nucleardepthdecoycheap>
8425 <entitydescription.nucleardepthdecoycheap>一种可通过深潜投放器部署的诱饵。投放后会下沉，然后产生一个明显的声信号，为潜艇转移注意力。由特定工程师廉价制造。</entitydescription.nucleardepthdecoycheap>
8426 <talenttree.politician>政客</talenttree.politician>
```

It's translation got mixed up with the talent politician.

non-talented nuclear depth decoy in chinese
![20240430211019_1](https://github.com/FakeFishGames/Barotrauma/assets/44329789/0fdc66f7-964d-4542-9b24-f679cf2f2261)

cheap version nuclear depth decoy in english
![20240430211045_1](https://github.com/FakeFishGames/Barotrauma/assets/44329789/0199a02c-19b9-42e9-bf03-acdf81e41320)

cheap version nuclear depth decoy in chinese
![20240430211006_1](https://github.com/FakeFishGames/Barotrauma/assets/44329789/d50d0497-1bb2-4e35-b645-ae1162378361)

### Reproduction steps

1.Set language to simplified chinese
2.Start level, go to fabricate machine
3.Search "核能", which means nuclear, can not find the cheap version nuclear depth decoy
4.Search "政客", which means politician, and able to find it.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.5.0 (Blood in the Water Update, hotfix 1)

### Which operating system did you encounter this bug on?

Windows

**Comment by AzureNocte** [comment]
And this issue also occurs within traditional chinese file.

**Comment by AzureNocte** [comment]
Tbh, the fact that big nuclear bomb being translated into politician is way hilarious.

**Comment by Regalis11** [comment]
Thank you for the report! I created a ticket about this in the issue tracker (https://github.com/FakeFishGames/Barotrauma/issues/13965), we'll look into it!

**Comment by Regalis11** [comment]
Fixed in the unstable build now, and the fix will be included in the next update.
